templates/Channel/Admin/Form/_additionalConfiguration.html.twig

```diff
@@ -1,3 +1,4 @@
 <div class="ui attached segment">
     {{ form_row(form.lowestPriceForDiscountedProductsVisible) }}
+    {{ form_row(form.lowestPriceForDiscountedProductsCheckingPeriod) }}
```

Comment:
Would be nice to have the form widget/row overridden to make it a right-labeled input with `days` as the label, similar to how it's done in price inputs